client/tailscale/localclient.go

```diff
@@ -1418,25 +1418,25 @@ func (lc *LocalClient) CheckUpdate(ctx context.Context) (*tailcfg.ClientVersion,
 	return &cv, nil
 }
 
-// TailFSSetFileServerAddr instructs TailFS to use the server at addr to access
+// DriveSetFileServerAddr instructs drive to use the server at addr to access
 // the filesystem. This is used on platforms like Windows and MacOS to let
-// TailFS know to use the file server running in the GUI app.
-func (lc *LocalClient) TailFSSetFileServerAddr(ctx context.Context, addr string) error {
+// drive know to use the file server running in the GUI app.
+func (lc *LocalClient) DriveSetFileServerAddr(ctx context.Context, addr string) error {
```

Comment:
What's the addr like? An IP:port? A URL?

Reply:
It's `ip:port`.